packages/expo-router/ios/LinkPreview/LinkPreviewNativeNavigation.swift

```diff
@@ -0,0 +1,55 @@
+import UIKit
+
+class LinkPreviewNativeNavigation {
+  private var preloadedView: UIView?
+  private var preloadedStackView: UIView?
+
+  init() {}
+
+  func pushPreloadedView() {
+    guard let preloadedView = preloadedView,
+      let preloadedStackView = preloadedStackView
```

Comment:
```suggestion
    guard let preloadedView,
      let preloadedStackView
```

a shorthand if the name is the same